PR: https://github.com/odoo/odoo/pull/

From: 045416474df6856166b70b0df806646115e96a18
From: Arnold Moyaux
Date: 2020-04-22 08:03:36

Structural Changes: 3
Total Changes: 6

[IMP] stock: orderpoint fileds rename and view improvements

There is a mix between qty and quantity. Set everything to quantity

Task: 2161378

================================= pseudo patch: =================================

--- a/addons/stock/models/stock_orderpoint.py
+++ b/addons/stock/models/stock_orderpoint.py
@@ -59,15 +59,15 @@ class StockWarehouseOrderpoint(models.Model):
         'uom.uom', 'Unit of Measure', related='product_id.uom_id')
     product_uom_name = fields.Char(string='Product unit of measure label', related='product_uom.display_name', readonly=True)
     product_min_qty = fields.Float(
-        'Minimum Quantity', digits='Product Unit of Measure', required=True, default=0.0,
+        'Min Quantity', digits='Product Unit of Measure', required=True, default=0.0,
         help="When the virtual stock equals to or goes below the Min Quantity specified for this field, Odoo generates "
              "a procurement to bring the forecasted quantity to the Max Quantity.")
     product_max_qty = fields.Float(
-        'Maximum Quantity', digits='Product Unit of Measure', required=True, default=0.0,
+        'Max Quantity', digits='Product Unit of Measure', required=True, default=0.0,
         help="When the virtual stock goes below the Min Quantity, Odoo generates "
              "a procurement to bring the forecasted quantity to the Quantity specified as Max Quantity.")
     qty_multiple = fields.Float(
-        'Qty Multiple', digits='Product Unit of Measure',
+        'Multiple Quantity', digits='Product Unit of Measure',
         default=1, required=True,
         help="The procurement quantity will be rounded up to this multiple.  If it is 0, the exact quantity will be used.")
     group_id = fields.Many2one(
